app/Http/Controllers/Admin/AdminHotelController.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace HotelBooking\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use HotelBooking\Http\Requests;
+use HotelBooking\Http\Controllers\Controller;
+use HotelBooking\AdminHotel;
+use DB;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class AdminHotelController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+
+    public function index()
+    {
+        $column = [
+            'id',
+            'hotel_id',
+            'username',
+            'name',
+            'email',
+            'phone',
+            ];
+        $adminHotels =  AdminHotel::with([
```

Comment:
make the code more readable by this way :
```   
 $with['getHotel'] =  function () {
        //some code here
    });
```
$adminHotels =  AdminHotel::with($with);